File: src/org/cappuccino_project/ide/intellij/plugin/psi/interfaces/ObjJResolveableElement.java
package org.cappuccino_project.ide.intellij.plugin.psi.interfaces;

import org.cappuccino_project.ide.intellij.plugin.stubs.interfaces.ObjJResolveableStub;

public interface ObjJResolveableElement<StubT extends ObjJResolveableStub> extends ObjJStubBasedElement<StubT> {
    boolean shouldResolve();
}

File: src/org/cappuccino_project/ide/intellij/plugin/psi/interfaces/ObjJResolveableElement.kt
package org.cappuccino_project.ide.intellij.plugin.psi.interfaces

import org.cappuccino_project.ide.intellij.plugin.stubs.interfaces.ObjJResolveableStub

interface ObjJResolveableElement<StubT : ObjJResolveableStub<*>> : ObjJStubBasedElement<StubT> {
    fun shouldResolve(): Boolean
}
